src/encoded/project/schema_views.py

```diff
@@ -0,0 +1,17 @@
+from snovault.project.schema_views import SnovaultProjectSchemaViews
+
+
+class SMaHTProjectSchemaViews(SnovaultProjectSchemaViews):
+
+    def get_submittable_item_names(self):
+        return []
+
+    def get_prop_for_submittable_items(self):
+        return 'submitted_id'
+
+    def get_properties_for_exclusion(self):
+        return ['date_created', 'submitted_by', 'uuid', 'schema_version',
+                'submission_centers', 'consortia', 'principals_allowed']
```

Comment:
I think these are good; some of them are redundant with the exclusion attributes, but that's fine.

Reply:
Status and accession as well perhaps? Though these are covered by the exclusion attributes as well, it may be more clear in error messaging if explicitly included here.